oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -769,6 +769,42 @@ void shouldValidateRequestObjectWhenJARValidationIsRequired() throws JOSEExcepti
             verify(requestObjectService).validateRequestObject(any());
         }
 
+        @Test
+        void shouldValidateRequestObjectWhenJARValidationIsNotRequired() throws JOSEException {
+            when(orchestrationAuthorizationService.isJarValidationRequired(any()))
+                    .thenReturn(false);
+            var keyPair = KeyPairHelper.GENERATE_RSA_KEY_PAIR();
+            var event = new APIGatewayProxyRequestEvent();
+            var jwtClaimsSet =
+                    new JWTClaimsSet.Builder()
+                            .audience("https://localhost/authorize")
+                            .claim("redirect_uri", REDIRECT_URI)
+                            .claim("response_type", ResponseType.CODE.toString())
+                            .claim("scope", SCOPE)
+                            .claim("state", STATE.getValue())
+                            .claim("nonce", NONCE.getValue())
+                            .claim("client_id", CLIENT_ID.getValue())
+                            .issuer(CLIENT_ID.getValue())
+                            .build();
```

Comment:
Small comment and unsure if it should be within this PR but this seems to repeated in multiple places within this file. Could this be pulled out ?